ui/src/shared/utils/labels.ts

```diff
@@ -0,0 +1,15 @@
+// Constants
+import {DEFAULT_LABEL_COLOR_HEX} from 'src/configuration/constants/LabelColors'
+
+// Types
+import {Label} from 'src/types/v2/labels'
+import {ILabel as LabelAPI} from '@influxdata/influx'
+
+export const addLabelDefaults = (l: LabelAPI): Label => ({
+  ...l,
+  properties: {
+    ...l.properties,
+    // add default color hex if missing
```

Comment:
Do we need this comment?

Reply:
We do this in the client and don't need to duplicate the work here